src/main/java/org/rumbledb/expressions/module/Prolog.java

```diff
@@ -24,22 +24,22 @@
 import org.rumbledb.exceptions.ExceptionMetadata;
 import org.rumbledb.expressions.Expression;
 import org.rumbledb.expressions.Node;
-import org.rumbledb.expressions.primary.FunctionDeclaration;
+import org.rumbledb.expressions.primary.InlineFunctionExpression;
 import sparksoniq.semantics.visitor.AbstractNodeVisitor;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class Prolog extends Expression {
 
-    private final List<FunctionDeclaration> functionDeclaration;
```

Comment:
This is a probably a typo from earlier. Should we rename this to functionDeclaration(s)